plugins/modules/foreman_compute_resource.py

```diff
@@ -71,39 +72,40 @@
 '''
 
 EXAMPLES = '''
-- name: vmware compute resource
+- name: create livirt compute resource
   foreman_compute_resource:
     name: example_compute_resource
     locations:
       - Munich
     organizations:
       - ATIX
-    provider: vmware
+    provider: libvirt
     provider_params:
-      url: vsphere.example.com
-      user: admin
-      password: secret
-      datacenter: ax01
+      url: libvirt.example.com
+      display_type: vnc
     server_url: foreman.example.com
     username: admin
     password: secret
     state: present
-
-- name: ovirt compute resource
+- name: update livirt compute resource
   foreman_compute_resource:
-    name: ovirt_compute_resource
+    name: example_compute_resource
+    description: updated compute resource
     locations:
-      - France/Toulouse
+      - Munich
     organizations:
-      - Example Org
-    provider: ovirt
+      - ATIX
+    provider: libvirt
     provider_params:
-      url: ovirt.example.com
-      user: ovirt-admin@example.com
-      password: ovirtsecret
-      datacenter: aa92fb54-0736-4066-8fa8-b8b9e3bd75ac
-      ovirt_quota: 24868ab9-c2a1-47c3-87e7-706f17d215ac
-      use_v4: true
+      url: libvirt.example.com
+      display_type: vnc
+    server_url: foreman.example.com
+    username: admin
+    password: secret
+    state: present
+- name: delete livirt compute resource
```

Comment:
missing empty line and this task has `state: present` while the name is "delete"? :)